library/src/lib/datetime-picker/datetime-picker.component.ts

```diff
@@ -58,6 +58,9 @@ export class DatetimePickerComponent implements OnInit, ControlValueAccessor {
     @Output()
     timeChange: EventEmitter<Date> = new EventEmitter<Date>();
 
+    @ViewChild(TimeComponent)
+    timeComponent;
```

Comment:
Can you think of a way to avoid this? I'd rather not slowly return to a convoluted datetime/date picker. Preferably, they'd only interact with inputs/events but sometimes I know its difficult to avoid so shrug

Reply:
We can't use the time component's inputs here because the displayed value for the hour does not automatically update when the model changes, and events won't work because we need to pass the value from parent to child and not child to parent.